### PR TITLE
[IMPAC-125] Fix/gulp build

### PR DESCRIPTION
**Pull request by xaun**
@ouranos I have merged your gulp work - please review

@cesar-tonnoir

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "main": [
     "dist/impac-angular.js",
-    "src/impac-angular.less"
+    "dist/impac-angular.less"
```
Comment (xaun) [inline bower.json]: @ouranos I think this is fine, but just confirming that parent apps less overrides, etc, will still function as they were when pointing to `src/impac-angular.less`? What's different about the new concatenated dist less file that allows us to do this now?

**Comment by xaun** [comment]
no idea why travis is breaking - will look at it in the morning.

**Comment by ouranos** [comment]
LGTM.  Please test in an MNOE project before merging :)

**Comment by xaun** [comment]
@cesar-tonnoir @ouranos tested on mnoe and good to merge :beers:

**Pull request merged**